packages/block-editor/src/components/link-control/README.md

```diff
@@ -59,6 +59,30 @@ The resulting default properties of `value` include:
 -   `title` (`string`, optional): Link title.
 -   `opensInNewTab` (`boolean`, optional): Whether link should open in a new browser tab. This value is only assigned when not providing a custom `settings` prop.
 
+Note that as `<LinkControl>` is often rendered within other components that may themselves re-render it is advisable that consumers memoize the `value` prop before passing it to the component. This avoids re-renders of LinkControl which may result in unwanted loss of
+changes users have may made when interacting with the control prior to the re-rendering.
```

Comment:
> This avoids re-renders of LinkControl which may result in unwanted loss of changes users have may made when interacting with the control prior to the re-rendering.

The memoization prevents unwanted sync with LinkControls' internal state, which is the cause of content loss. The component will still re-render every time the parent re-renders.

Reply:
Thanks for re-clarifying this.

Reply:
I've updated but I'd be grateful for any re-wording suggestions.